### PR TITLE
Update Table 4 Numbering systems

### PR DESCRIPTION
**Pull request by FrankYFTang**
Add more numbering systems which on CLDR 37

Close https://github.com/tc39/ecma402/issues/437

### CODE REVIEW

**Comment by sffc** [comment]
Would be good to get sign-off from the browser vendors.

@jswalden @litherum

**Comment by FrankYFTang** [comment]
@rkirsling

**Comment by FrankYFTang** [comment]
The list I proposed here (except "diak" and "segment") are ALREADY supported by two browser engines already:

* Safari version 13.1
* Chrome / v8 m81

"diak" and "segment" are new in CLDR / ICU and Chrome will support them in m84 with the update to ICU 67.1 .

**Comment by rkirsling** [comment]
Sounds good to me. JSC is aiming to defer to ICU / CLDR directly as much as possible, even if our pace of updating is slower than than V8's (due to reliance on system ICU where available).

**Comment by leobalter** [comment]
Let's bring this up for the meeting this week.

**Comment by sffc** [comment]
This has ECMA-402 consensus.  We would like to see a more regular process of updating this list, but that doesn't block the current PR.  We need to achieve TC39 consensus before it can be merged.

**Comment by anba** [comment]
So we don't expect any compatibility issues with other software (https://github.com/tc39/ecma402/issues/437#issuecomment-632054710)?

File: spec/numberformat.html [hunk 1]
```diff
@@ -390,6 +550,10 @@ <h1>PartitionNotationSubPattern ( _numberFormat_, _x_, _n_, _exponent_ )</h1>
             <td>telu</td>
             <td>U+0C66 to U+0C6F</td>
           </tr>
+          <tr>
+            <td>tirh</td>
```
Comment (anba) [inline spec/numberformat.html:554]: Nit: Move `tirh` after `tibt`.

Comment (FrankYFTang) [reply spec/numberformat.html]: done

**Comment by sffc** [comment]
Is this PR considered editorial or normative?  Is it normative because conforming implementations could become non-conforming based on this PR, or is it editorial because it reflects web reality?

**Comment by FrankYFTang** [comment]
> Is this PR considered editorial or normative? Is it normative because conforming implementations could become non-conforming based on this PR, or is it editorial because it reflects web reality?

It is **normative** because https://ecma-international.org/ecma-402/#sec-partitionnumberpattern stated in step 8.e.iii.3 
"If the numberFormat.[[NumberingSystem]] matches one of the values in the "Numbering System" column of Table 3 below, then"

**Comment by FrankYFTang** [comment]
So.. TC39 reach consensus this morning. @sffc @leobalter  @gibson042  could you approve?

**Pull request merged**